src/Components/ImagesTable/ImageLink.js

```diff
@@ -35,9 +38,9 @@ const ProvisioningLink = ({ imageId, isExpired, isInClonesTable }) => {
   );
 
   const provider = getImageProvider(image);
-  if (!error && image.share_with_sources) {
+  if (!error) {
```

Comment:
I would prefer to keep this actually. I think we should only link to provisioning if the image was shared with a source.

Reply:
this was extracted to a "allowed for provider" function.

Reply:
missed that, sorry!